Makefile

```diff
@@ -387,20 +405,44 @@ assets_nocore: \
 	ebpf_nocore
 	$(CMD_GO) run github.com/shuLhan/go-bindata/cmd/go-bindata $(IGNORE_LESS52) -pkg assets -o "assets/ebpf_probe.go" $(wildcard ./user/bytecode/*.o)
 
+
+$(TARGET_LIBPCAP):
+	cd lib/libpcap && \
+		CC=$(CC) ./configure --disable-rdma --disable-shared --disable-usb \
+			--disable-netmap --disable-bluetooth --disable-dbus --without-libnl \
+			--without-dpdk --without-dag --without-septel --without-snf \
+			--without-turbocap --host=$(LIBPCAP_ARCH) && \
+		make && make install
```

Comment:
Github Action run failed  :  https://github.com/gojue/ecapture/actions/runs/7960865425/job/21730956179?pr=478#step:6:550

```shell
[ -d /usr/local/lib ] || \
      (mkdir -p /usr/local/lib; chmod 755 /usr/local/lib)
  /usr/bin/install -c -m 644 libpcap.a /usr/local/lib/libpcap.a
  /usr/bin/install: cannot create regular file '/usr/local/lib/libpcap.a': Permission denied
```

Do you need to add `sudo`?

Reply:
added